kafka_consumer/tests/conftest.py

```diff
@@ -57,12 +70,6 @@ def dd_environment(mock_local_kafka_hosts_dns):
             WaitFor(create_topics, attempts=60, wait=3),
             WaitFor(initialize_topics),
         ],
-        env_vars={
-            # Advertising the hostname doesn't work on docker:dind so we manually
-            # resolve the IP address. This seems to also work outside docker:dind
-            # so we got that goin for us.
-            'KAFKA_HOST': HOST_IP,
```

Comment:
Was this used anywhere? This may be a env var that the test environment uses indirectly?

Reply:
It was used in the `advertised listeners` properties that we no longer use since Fanny switched to the bitnami images